### ISSUE TITLE
Enable the download of raw data files

### ISSUE THREAD

**Issue by dalonsoa** [ImperialCollegeLondon/paricia#334]
The file cannot be downloaded because Django in production, by default, prevents from downloading raw data files for security reasons. They need to be served in a different way. If downloading the raw data files is a thing, I'll create an issue about that.

_Originally posted by @dalonsoa in https://github.com/ImperialCollegeLondon/paricia/issues/332#issuecomment-2362803578_

**Comment by ICHydro** [comment]
There is no need for the user to download the raw data file. In fact, there is probably no need to keep the raw file after the data have been processed. Similarly, there is probably no need for the user to be able to change the data file of the import after it is created; they can just create a new import.

**Comment by dalonsoa** [comment]
If the raw data file is not to be kept, is there any reason for not deleting the whole data import object altogether once the import is successful? I don’t think it has much use if there’s no file.

**Comment by ICHydro** [comment]
Indeed, I am reconsidering this. There is probably value in keeping the file to enable future reprocessing, and it can also serve as a place for users to keep the raw data files to ensure that they do not get lost, which enhances reproducibility and traceability. So let's keep this as an enhancement, including the possibility for users to download the original file.

**Comment by dalonsoa** [comment]
If we are going to keep the raw data and enable the user to download it, we might want to consider using a proper external service for this rather than storing the data in the VM, something like AWS S3 buckets or Azure Blob storage or something along those lines. 

@ICHydro I'd suggest we leave this on hold until we can discuss the options and implications.

**Comment by dalonsoa** [comment]
@ICHydro For what @stephwills has told me, you are considering moving the tool to Azure. As part of the wider VIEW project, it might be a good time to discuss the storage and download of the original raw files, as well as storing the data in a "proper" database in Azure, rather than in a database within the VM, which is what we have done so far.

Nothing to do right now or to rush through, but something we should definitely consider in the near term. Might be worth including it as a discussion point for the next kick-off meeting.

**Comment by dalonsoa** [comment]
Also pinging @AdrianDAlessandro , to be in the loop.
